ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -2786,6 +2786,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-zones-tp-f28
```

Comment:
Also need a destructive test ci job.

Reply:
ok.

Reply:
also need to add them in 4.16?

Reply:
it's tp in 4.15, 4.16 should be enabled by default. so i think no need.

Reply:
We can not assume the 4.16 case, if 4.16 implement that, then update the ci jobs accordingly on that time, one on e can remember this story if 4.16 did not implement this on the plan.

Reply:
ok, done. so if it implement in 4.16 as plan. what i need to do in the future is remove the tag-->FEATURE_SET: TechPreviewNoUpgrade.  right?

Reply:
There is already an existing `vsphere-ipi-zones` ci job, once it is implemented in 4.16, you need to drop the tp job and update `REQUIRED_OCP_VERSION` to 4.16

Reply:
ok.